src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -1296,6 +1298,16 @@ synchronized public void addCategoryToFile(Bot bot, Category c) {
       error(e);
     }
   }
+  
+  boolean sleep = false;

```

Comment:
member variables should go to the top of the class.

Reply:
agreed fixed